lib/api/1.1/northbound/workflows.js

```diff
@@ -104,34 +104,24 @@ function workflowsRouterFactory (
     }));
 
     /**
-     * @api {get} /api/1.1/workflows/library GET /library
+     * @api {get} /api/1.1/workflows/library/:injectableName GET /library/:injectableName
      * @apiVersion 1.1.0
-     * @apiDescription get list of workflows possible to run
-     * @apiName workflows-library
-     * @apiSuccess {json} workflows list of workflows that are possible to run
-     * @apiGroup workflows
-     */
-
-    router.get('/workflows/library', rest(function () {
-        return workflowApiService.getGraphDefinitions();
-    }));
-
-    /**
-     * @api {get} /api/1.1/workflows/library/:identifier GET /library/:identifier
-     * @apiVersion 1.1.0
-     * @apiDescription get a specific workflow from the library
-     * @apiParam {String} identifier of workflow, must cast to ObjectId
+     * @apiDescription get a specific workflow definition from the library
+     * @apiParam {String} injectableName of workflow definition
      * @apiName workflows-library-item
-     * @apiSuccess {json} workflow the workflow that have the <code>identifier</code>
+     * @apiSuccess {json} workflow the workflow that have the <code>injectableName</code>
      * @apiGroup workflows
      */
 
-    router.get('/workflows/library/:identifier', rest(function (req) {
-        return workflowApiService.getGraphDefinitions(req.params.identifier)
+    router.get('/workflows/library/:injectableName', rest(function (req) {
+        if (req.params.injectableName === '*') {
```

Comment:
probably should be two routes:  GET /workflows/library and GET /workflows/library/:injectableName